app/src/main/java/com/onboarding/parkingsystemkotlin/mvp/view/DateTimePickerView.kt

```diff
@@ -2,13 +2,35 @@ package com.onboarding.parkingsystemkotlin.mvp.view
 
 import androidx.fragment.app.DialogFragment
 import com.onboarding.parkingsystemkotlin.databinding.DateTimePickerReservationFragmentBinding
+import com.onboarding.parkingsystemkotlin.listener.DateTimeListener
 import com.onboarding.parkingsystemkotlin.mvp.contract.DateTimePickerContract
 import com.onboarding.parkingsystemkotlin.mvp.view.base.FragmentView
+import java.util.*
```

Comment:
do not use wildcard imports pls

Reply:
every time I used a Calendar on a new file, Kotlin imported java.util.*  that one must've slipped trough

Reply:
you can configure the imports in the AS preferences, lets check it later, please remind me